fontc_crater/src/args.rs

```diff
@@ -13,33 +18,21 @@ pub(super) struct Args {
 
 #[derive(Debug, Subcommand, PartialEq)]
 pub(super) enum Commands {
-    Compile(RunArgs),
-    Diff(RunArgs),
-    Report(ReportArgs),
     Ci(CiArgs),
 }
 
 #[derive(Debug, PartialEq, clap::Args)]
-pub(super) struct RunArgs {
+pub(super) struct CiArgs {
+    /// Path to a json list of repos + revs to run.
+    pub(super) to_run: PathBuf,
     /// Directory to store font sources and the google/fonts repo.
     ///
     /// Reusing this directory saves us having to clone all the repos on each run.
     ///
     /// This directory is also used to write cached results during repo discovery.
-    #[arg(short, long, default_value = "~/.fontc_crater_cache")]
-    pub(super) cache_dir: PathBuf,
-    /// Optional path to write out results (as json)
-    #[arg(short = 'o', long = "out")]
-    pub(super) out_path: Option<PathBuf>,
-    /// for debugging, execute only a given number of fonts
-    #[arg(long)]
-    pub(super) limit: Option<usize>,
-}
+    #[arg(short, long = "cache")]
+    cache_dir: Option<PathBuf>,
```

Comment:
Why did we remove the default value from here? - now it won't show up in `--help`

Reply:
If we have to keep the env var lets document it and the interaction with this flag here

Reply:
added a note about the default to the help docs.

As I think you inferred this was removed so that we could check for `None` and then use the env var if it's set.